srcpkgs/hplip-plugin/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'hplip-plugin'
+# Must be in sync with main hplip package
+pkgname=hplip-plugin
+version=3.22.6
+revision=1
+archs="i686 x86_64 armv6h armv7h aarch64"
```

Comment:
armv6h and armv7h are not valid archs

Reply:
armv6hf and armv7hf are also not valid archs

Reply:
Corrected

Reply:
still not correct, look at other templates or to common/build-profiles, the correct arch names are armv6l and armv7l